makefiles/info-global.inc.mk

```diff
@@ -28,6 +28,9 @@ define board_missing_features
 
   include $(RIOTBASE)/Makefile.dep
 
+  # Include provided features from drivers after all dependencies are known
+  include $(RIOTBASE)/drivers/Makefile.features
```

Comment:
If it is "features" it should be included in `Makefile.features`.

Reply:
The features provided by (radio) drivers must already know the driver modules used in the application. Because of this `Makefile.features` has to be included after `Makefile.dep`. I don't know if we want this or not.
If I put the `drivers/Makefile.features` file in the global `Makefile.features`, I have to change a few things in Makefile.include that break the build. This is still unclear for me why.

Reply:
Indeed, I found why it does not work. I will explain in the main comments.